web/modeshape-webdav/src/main/java/org/modeshape/webdav/methods/AbstractMethod.java

```diff
@@ -116,32 +123,11 @@ protected AbstractMethod() {
     }
 
     public static String lastModifiedDateFormat( final Date date ) {
-        DateFormat df = thLastmodifiedDateFormat.get();
-        if (df == null) {
-            df = new SimpleDateFormat(LAST_MODIFIED_DATE_FORMAT, Locale.US);
-            df.setTimeZone(TimeZone.getTimeZone("GMT"));
-            thLastmodifiedDateFormat.set(df);
-        }
-        return df.format(date);
+        return LAST_MODIFIED_DATE_FORMAT.format(date);
     }
 
     public static String creationDateFormat( final Date date ) {
-        DateFormat df = thCreationDateFormat.get();
-        if (df == null) {
-            df = new SimpleDateFormat(CREATION_DATE_FORMAT);
-            df.setTimeZone(TimeZone.getTimeZone("GMT"));
-            thCreationDateFormat.set(df);
-        }
-        return df.format(date);
-    }
-
-    public static String getLocalDateFormat( final Date date,
-                                             final Locale loc ) {
-        DateFormat df = thLocalDateFormat.get();
-        if (df == null) {
-            df = new SimpleDateFormat(LOCAL_DATE_FORMAT, loc);
-        }
-        return df.format(date);
+        return CREATION_DATE_FORMAT.format(date);
```

Comment:
`SimpleDateFormat` is not thread safe, so you can't share it across threads. The static instance _can_ be used across threads since this method is static, so it's not safe to make this change.